internal/pkg/describe/uri.go

```diff
@@ -412,7 +414,7 @@ func (u *accessURI) strings() []string {
 		if !strings.HasPrefix(u.Path, "/") {
 			path = fmt.Sprintf("/%s", u.Path)
 		} else if u.Path != "/" {
```

Comment:
tiny nit: we probably don't even need the `if u.Path != "/"` because even if it is, `path = re.ReplaceAllString(u.Path, "/")` will still yield "/"